samples/matter/lock/src/app_task.cpp

```diff
@@ -636,22 +638,28 @@ void AppTask::LockStateChanged(BoltLockManager::State state, BoltLockManager::Op
 	case BoltLockManager::State::kLockingInitiated:
 		LOG_INF("Lock action initiated");
 		sLockLED.Blink(50, 50);
+#ifdef CONFIG_CHIP_NUS
+		GetNUSService().SendData("locking", sizeof("locking"));
+#endif
 		break;
 	case BoltLockManager::State::kLockingCompleted:
 		LOG_INF("Lock action completed");
 		sLockLED.Set(true);
 #ifdef CONFIG_CHIP_NUS
-		GetNUSService().SendData("Locked", sizeof("Locked"));
+		GetNUSService().SendData("locked", sizeof("locked"));
```

Comment:
BTW. NUS API would be more concise if the user didn't have to provide the size of a command. Not sure if we can assume null terminated string each time though.